oneview/resource_server_profile_template.go

```diff
@@ -2880,7 +2891,15 @@ func resourceServerProfileTemplateUpdate(d *schema.ResourceData, meta interface{
 						Name:              logicalDrivesItem["name"].(string),
 						NumPhysicalDrives: logicalDrivesItem["num_physical_drives"].(int),
 						NumSpareDrives:    logicalDrivesItem["num_spare_drives"].(int),
-						SasLogicalJBODId:  logicalDrivesItem["sas_logical_jbod_id"].(int),
+					}
+					if logicalDrivesItem["sas_logical_jbod_id"].(string) != "" {
```

Comment:
create and update follow similar piece of code...can't we put it into a function and call it in both the place?